src/docker/docker-compose.yml

```diff
@@ -84,7 +84,13 @@ services:
       - ../streamlit/config.toml:/root/gak/config.toml
       - ../streamlit/credentials.toml:/root/gak/credentials.toml
     healthcheck:
-      test: ["CMD", "curl", "-Lf", "http://localhost:8501/${BASE_PATH}healthz"]
+      test:
+        [
```

Comment:
@rjurney - curious what you are using that is making these formatting changes? Never been a big fan of fixing things that aren't broken.  I don't think there's any linter or anything on this repo, so curious what this is for?  Just a styling preference or some special editor/plugin you're using?